packages/back-end/src/models/WebhookSecretModel.ts

```diff
@@ -47,6 +47,10 @@ export class WebhookSecretDataModel extends BaseClass {
     });
   }
 
+  public async findByKey(key: string) {
+    return this._findOne({ key });
+  }
```

Comment:
I'd prefer not exposing secret values outside of the model.  It looks like this is only used to check if a key exists so we can delete it.  Instead, let's just make a `deleteByKey` method that does this internally.